app/opinions/getNewestOpinions.js

```diff
@@ -0,0 +1,42 @@
+import express from "express";
+import bodyParser from "body-parser";
+import mongoose from "mongoose";
+
+const app = express();
+
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  }),
+);
+app.use(express.static("public"));
+
+mongoose.connect("mongodb://localhost:27017/Opinions-db", {
+  useNewUrlParser: true,
+});
+
+const OpinionSchema = new mongoose.Schema({
+  rating: Number,
+  description: String,
+  name: String,
+  city: String,
+});
+
+const Opinions = mongoose.model("Opinions", OpinionSchema);
+
+app.get("/", function (req, res) {
```

Comment:
I think it would be better to put every endpoint into routes file. You should also use `express.Router()` instead of `express()` and then `app.use("/opinions", opinionsRouter)`